bundler/spec/remove/install_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+RSpec.describe "bundle install" do
+  before :each do
+    gemfile <<-G
+      source "#{file_uri_for(gem_repo1)}"
+      gem "rack"
+    G
+  end
+
+  describe "with --install", :bundler => 2 do
+    it "shows a deprecation warning" do
+      bundle "remove rack --install"
+
+      expect(err).to include "[DEPRECATED] The `--install` flag has been deprecated. `bundle install` is triggered by default."
+    end
+  end
```

Comment:
Can you move this spec to `spec/other/major_deprecation_spec.rb`. I like to keep all deprecation specs together.